pylint/extensions/_check_docs_utils.py

```diff
@@ -210,8 +211,9 @@ def __init__(self, doc):
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__}:'''{self.doc}'''>"
 
-    def is_valid(self):
-        return False
+    def matching_sections(self) -> int:
```

Comment:
```suggestion
    def is_valid(self) -> bool:
        return self.matching_sections() > 0

    def matching_sections(self) -> int:
```
To avoid the breaking change ?

Reply:
I thought about this, but then the method name doesn't match its behaviour..

However, this is an extension so is this actually part of the public API?

Reply:
Yeah you're right it would not be what someone expect from "is_valid". Let's consider it's not a public API.